x-pack/test/reporting_api_integration/reporting_and_security/usage/api_counters.ts

```diff
@@ -92,39 +92,53 @@ export default function ({ getService }: FtrProviderContext) {
       });
 
       it('downloading', async () => {
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/kraz0qle154g0763b569zz83')
-          .auth('test_user', 'changeme');
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/kraz0vj4154g0763b5curq51')
-          .auth('test_user', 'changeme');
-        await supertestUnauth
-          .get('/api/reporting/jobs/download/k9a9rq1i0gpe1457b17s7yc6')
-          .auth('test_user', 'changeme');
+        try {
+          await Promise.all([
+            supertestUnauth
+              .get('/api/reporting/jobs/download/kraz0qle154g0763b569zz83')
+              .auth('test_user', 'changeme'),
+            supertestUnauth
+              .get('/api/reporting/jobs/download/kraz0vj4154g0763b5curq51')
+              .auth('test_user', 'changeme'),
+            supertestUnauth
+              .get('/api/reporting/jobs/download/k9a9rq1i0gpe1457b17s7yc6')
+              .auth('test_user', 'changeme'),
+          ]);
+        } catch (error) {
+          log.error(error);
+        }
 
+        log.info(`waiting on internal stats aggregation...`);
         await waitOnAggregation();
+        log.info(`waiting on aggregation completed.`);
 
+        log.info(`calling getUsageStats...`);
         expect(
           getUsageCount(await usageAPI.getUsageStats(), `get /api/reporting/jobs/download/{docId}`)
         ).to.be(3);
       });
 
       it('deleting', async () => {
-        await supertestUnauth
-          .delete('/api/reporting/jobs/delete/krazcyw4156m0763b503j7f9')
-          .auth('test_user', 'changeme')
-          .set('kbn-xsrf', 'xxx');
-
-        await supertestUnauth
-          .delete('/api/reporting/jobs/delete/krazaxch156m0763b5bf81ov')
-          .auth('test_user', 'changeme')
-          .set('kbn-xsrf', 'xxx');
+        log.info(`sending 1 delete request...`);
+
+        try {
+          await supertestUnauth
+            .delete('/api/reporting/jobs/delete/krazcyw4156m0763b503j7f9')
+            .auth('test_user', 'changeme')
+            .set('kbn-xsrf', 'xxx');
```

Comment:
Using a single report job in this test, and wrapping the request in try/catch seemed to resolve the issues with this test consistently failing.